packages/core/demo/demo-data/bar.ts

```diff
@@ -107,7 +107,8 @@ export const simpleBarOptions = {
 		}
 	},
 	legendClickable: true,
-	containerResizable: true
+	containerResizable: true,
+	maxWidth: 50
```

Comment:
Is this for each bar?

This should become:
```js
{
   bars: {
	  maxWidth: 50
   }
}
```

Reply:
sure